ci/python-publish.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install setuptools wheel twine
+        pip install -r requirements.txt
```

Comment:
I might be misunderstanding, but there isn't anything that requires `requirements.txt` to exist is there? As such this may fail for some users. In other workflows we've guarded this to check that the file exists:

`if [ -f requirements.txt ]; then pip install -r requirements.txt; fi`

Would that make sense here?

Reply:
Oh yes, definitely. Made the change.